pkg/noderesourcetopology/plugin.go

```diff
@@ -64,7 +59,7 @@ type PolicyHandlerMap map[topologyv1alpha1.TopologyManagerPolicy]tmScopeHandler
 
 // TopologyMatch plugin which run simplified version of TopologyManager's admit handler
 type TopologyMatch struct {
-	nodeResTopologyPlugin
+	lister              *listerv1alpha1.NodeResourceTopologyLister
```

Comment:
I think this, being an interface, should be passed around as value, not as pointer

Reply:
Ack